src/hotspot/cpu/aarch64/assembler_aarch64.hpp

```diff
@@ -3353,6 +3353,38 @@ void mvnw(Register Rd, Register Rm,
   INSN(sve_whilelsw, 0b111, 0);
 #undef INSN
 
+  void encode_cvtf_T (SIMD_RegVariant& T_dst, SIMD_RegVariant& T_src) {
```

Comment:
It's better to be a non-public function.

Reply:
Yes, I will fix it.